rllib/offline/offline_policy_evaluation_runner.py

```diff
@@ -613,7 +619,7 @@ def _batch_iterator(self) -> MiniBatchRayDataIterator:
         return self.__batch_iterator
 
     @property
-    def _device(self) -> DeviceType:
+    def _device(self) -> Union[DeviceType, None]:
```

Comment:
same here. @simonsays1980 please advise if `None` is allowed

Reply:
See my comment above.